qt/python/mantidqt/mantidqt/widgets/superplot/presenter.py

```diff
@@ -199,6 +202,25 @@ def _sync_with_current_plot(self):
             self._model.add_workspace(ws_name)
             self._model.add_data(ws_name, spec_index)
 
+    def get_kwargs_from_settings(self):
+        """
+        Get the useful plot keywork arguments from the global settings.
+
+        Returns:
+            dict(str: str): plot keywork arguments
```

Comment:
Couple of small typos:
```suggestion
        Get the useful plot keyword arguments from the global settings.

        Returns:
            dict(str: str): plot keyword arguments
```

Reply:
Done.